templates/3dsmax/scripts/3dsmax.ps1

```diff
@@ -162,23 +98,40 @@ if ($renderer -eq "arnold")
     $pre_render_script_content += "renderMessageManager.LogDebugMessage = true`r`n"
 }
 
-if ($renderer -like "vray")
+if ($renderer -like "vray*")
 {
-    Write-Host "3ds Max is using the Vray renderer " 
+    Write-Host "3ds Max is using the Vray renderer: $renderer" 
     $outputPath = "$env:AZ_BATCH_TASK_WORKING_DIR\images\" -replace "\\", "\\"
-    $pre_render_script_content += "-- Set output channel path`r`n"
-    $pre_render_script_content += "rendererName = r as string`r`n"
-    $pre_render_script_content += "indexVrayAdv = findString rendererName ""V_Ray_Adv_""`r`n"
-    $pre_render_script_content += "if (indexVrayAdv == 1 and r.output_splitgbuffer and r.output_splitfilename != """") then (fileName = ""$outputPath"" + (filenameFromPath r.output_splitfilename); r.output_splitfilename = fileName)`r`n"
-    $pre_render_script_content += "if (indexVrayAdv == 1 and r.output_saveRawFile and r.output_rawFileName != """") then (fileName = ""$outputPath"" + (filenameFromPath r.output_rawFileName); r.output_rawFileName = fileName)`r`n"
-
-    $pre_render_script_content += "indexVrayNext = findString rendererName ""V_Ray_Next_""`r`n"
-    $pre_render_script_content += "if (indexVrayNext == 1 and r.output_splitgbuffer and r.output_splitfilename != """") then (fileName = ""$outputPath"" + (filenameFromPath r.output_splitfilename); r.output_splitfilename = fileName)`r`n"
-    $pre_render_script_content += "if (indexVrayNext == 1 and r.output_saveRawFile and r.output_rawFileName != """") then (fileName = ""$outputPath"" + (filenameFromPath r.output_rawFileName); r.output_rawFileName = fileName)`r`n"
-
-    $pre_render_script_content += "indexVrayNextGpu = findString rendererName ""V_Ray_GPU_Next_""`r`n"
-    $pre_render_script_content += "if (indexVrayNextGpu == 1 and r.V_Ray_settings.output_splitgbuffer and r.V_Ray_settings.output_splitfilename != """") then (fileName = ""$outputPath"" + (filenameFromPath r.V_Ray_settings.output_splitfilename); r.V_Ray_settings.output_splitfilename = fileName)`r`n"
-    $pre_render_script_content += "if (indexVrayNextGpu == 1 and r.V_Ray_settings.output_saveRawFile and r.V_Ray_settings.output_rawFileName != """") then (fileName = ""$outputPath"" + (filenameFromPath r.V_Ray_settings.output_rawFileName); r.V_Ray_settings.output_rawFileName = fileName)`r`n"
+    $vrayLogFile = "$env:AZ_BATCH_TASK_WORKING_DIR\VRayLog.log" -replace "\\", "\\"
+
+    $pre_render_script_content += "r.system_vrayLog_level = 4`r`n"
+    $pre_render_script_content += "r.system_vrayLog_file = ""$vrayLogFile""`r`n"
```

Comment:
In previous version - without DR activated - vray log file was located at its default location `$env:LOCALAPPDATA\Temp\vraylog.txt`

1. This pattern is written in job templates
```
{
                            "filePattern": "vraylog.*",
                            "destination": {
                                "autoStorage": {
                                    "fileGroup": "[parameters('outputs')]",
                                    "path": "[parameters('jobName')]/logs/frame_{0}"
                                }
                            },
                            "uploadOptions": {
                                "uploadCondition": "taskCompletion"
                            }
                        }
```

2. Then the default file location is copied at the end of the 3dsmax.ps1 script
```
Copy-Item "$env:LOCALAPPDATA\Temp\vraylog.txt" . -ErrorAction SilentlyContinue
```